Shell/InterpretedNormalizer.cpp

```diff
@@ -404,8 +404,9 @@ Formula* InterpretedNormalizer::NLiteralTransformer::transform(Formula* f)
 // InterpretedNormalizer
 //
 
-InterpretedNormalizer::InterpretedNormalizer()
+InterpretedNormalizer::InterpretedNormalizer(InequalityNormalizer* norm)
 : _litTransf(new NLiteralTransformer())
```

Comment:
Not saying it's bad, but do you agree that a non-null `InequalityNormalizer` actually never passed to `InterpretedNormalizer`? (Maybe in tests? I only see a single use in `Preprocess.cpp` on line 89.)

Reply:
good spot. that was indeed dead code. removed it.